### PR TITLE
fix: import nuxt composables from #imports

### PR DESCRIPTION
**Pull request by danielroe**
This is a DX improvement when developing - we can avoid loading the entire barrel file at `#app` by using the new granular imports merged in https://github.com/nuxt/nuxt/pull/23951.

Checklist:
- [ ] issue number linked above after pound (`#`)
    - replace "Closes " with "Contributes to" or other if this PR does not close the issue
- [ ] manually checked my feature / checking not applicable
- [ ] wrote tests / testing not applicable
- [ ] attached screenshots / screenshot not applicable

### CODE REVIEW

File: src/runtime/composables/local/useAuthState.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { computed, watch, ComputedRef } from 'vue'
-import { CookieRef } from '#app'
+import { CookieRef } from '#imports'
```
Comment (danielroe) [inline src/runtime/composables/local/useAuthState.ts:2]:
```suggestion
import type { CookieRef } from '#app'
```

**Comment by zoey-kaiser** [comment]
Hi @danielroe 👋 

Thanks for your PR. I will pass this onto @BracketJohn for a review!

**Review by DraftProducts — APPROVED** [review]

**Comment by zoey-kaiser** [comment]
Quick update on this: Before merging this PR, we will need to upgrade our Nuxt version from `3.4.2` to at least `3.8.1`, in which this feature was released. 

Once I merge the Nuxt upgrade, I will then move forward with merging this PR too! 🤗

**Comment by danielroe** [comment]
This PR should be backwards-compatible. It will only result in a perf benefit from 3.8.1 onwards though.

**Comment by zoey-kaiser** [comment]
> This PR should be backwards-compatible. It will only result in a perf benefit from 3.8.1 onwards though.

That is very cool! Ill pull this branch then and test it! Thanks for all of your input @danielroe ❤️

**Comment by danielroe** [comment]
done ✅

**Review by zoey-kaiser — APPROVED** [review]

**Pull request merged**